frontend/src/metabase/query_builder/components/expressions/ExpressionMode.js

```diff
@@ -0,0 +1,50 @@
+import "brace/mode/java";
+
+class ExpressionHighlight extends window.ace.acequire(
+  "ace/mode/text_highlight_rules",
+).TextHighlightRules {
+  constructor() {
+    super();
+
+    this.$rules = {
+      start: [
+        {
+          token: "string",
```

Comment:
I think it's going to slightly bork, when people are using regex - probably not a huge problem - example:
![image](https://user-images.githubusercontent.com/1447303/144909975-3b2a2f03-9013-4d4a-beb9-1488ef54b21f.png)

Reply:
Likewise!